src/Domain/Strava/Activity/ActivityIntensity.php

```diff
@@ -5,36 +5,73 @@
 namespace App\Domain\Strava\Activity;
 
 use App\Domain\Strava\Athlete\AthleteRepository;
+use App\Domain\Strava\Ftp\EFtpRepository;
 use App\Domain\Strava\Ftp\FtpRepository;
 use App\Infrastructure\Exception\EntityNotFound;
 
-final readonly class ActivityIntensity
```

Comment:
This class need to be kept read only. Any service should be injected into this class via the constructor